e2e/test-component/scenarios/embedding-sdk/interactive-question.cy.spec.tsx

```diff
@@ -20,6 +20,7 @@ describe("scenarios > embedding-sdk > interactive-question", () => {
         idAlias: "sqlQuestionId",
       },
     );
+
```

Comment:
Can we remove this? I don't mind the formatting if we have other change  in the file, but this...